rotkehlchen/db/dbhandler.py

```diff
@@ -2178,6 +2178,28 @@ def save_balances_data(self, data: Dict[str, Any], timestamp: Timestamp) -> None
         self.add_multiple_balances(balances)
         self.add_multiple_location_data(locations)
 
+    def add_known_location(self, location: Location) -> None:
```

Comment:
Generally this is confusing me a lot. Can you please explain what these locations are? From the code it seems to be the locations for which we have data imported from external sources.

`ConnectedExchanges` is then wrong as a name. `KnownLocations` also.

If they are what I think they are the name should be: `ExternallyImportedLocations`.

Please either use that name or propose a new one. And use it everywhere you currently use `known_location`.